org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -30,6 +30,153 @@ Parameters:
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref bridgeProdAdminGroup
+    permissionSetName: 'Administrator'
+    managedPolicies: [ 'arn:aws:iam::aws:policy/AdministratorAccess' ]
+    sessionDuration: 'PT12H'
+
+SsoBridgeDevDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-bridgedev-developer'
+  StackDescription: 'SSO: developer role used by bridgedev developer group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref BridgeDevAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref bridgeDevDeveloperGroup
+    permissionSetName: 'Developer'
+    managedPolicies: [ 'arn:aws:iam::aws:policy/PowerUserAccess' ]
+    sessionDuration: 'PT12H'
+
+SsoBridgeProdDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-bridgeprod-developer'
+  StackDescription: 'SSO: developer role used by bridgeprod developer group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref BridgeProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref bridgeProdDeveloperGroup
+    permissionSetName: 'Developer'
+    managedPolicies: [ 'arn:aws:iam::aws:policy/PowerUserAccess' ]
+    sessionDuration: 'PT12H'
+
+SsoBridgeIosDeveloperPolicy:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/IAM/managed-policy.yaml
+  StackName: sso-bridge-ios-developer-policy
+  Parameters:
+    PolicyDocument: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Sid": "ListBucketAccess",
+            "Action": [ "s3:ListAllMyBuckets" ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:s3:::*"
+          },
+          {
+            "Sid": "LocationBucketAccess",
+            "Action": [ "s3:ListBucket", "s3:GetBucketLocation" ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:s3:::ios-apps.sagebridge.org"
+          },
+          {
+            "Sid": "BucketObjectAccess",
+            "Action": [ "s3:PutObject", "s3:PutObjectAcl", "s3:GetObject", "s3:GetObjectAcl", "s3:DeleteObject" ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:s3:::ios-apps.sagebridge.org/*"
+          }
+        ]
+      }
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref MasterAccount
```

Comment:
This target should be a bridge acct

Reply:
fixed by replacing this block with `inlinePolicy` in SsoBridgeProdIosDeveloper